infra/notifications/src/app.js

```diff
@@ -178,6 +182,9 @@ app.post('/events', async (req, res) => {
     event.blockNumber
   } logIndex=${event.logIndex}`
 
+  // Return 200 to the event-listener without waiting for processing of the event.
```

Comment:
That means the listener won't retry in case of a failure. It's probably ok for now since anyway email/mobile push  delivery is not guaranteed.

I think in longer term we should move to a queue model where the listener enqueues tasks that get consumed by the downstream services as opposed to using webhook. This gives more flexibility to the services to decide what to do in case of failure (let item in the queue for re-processing or not).

Reply:
Yep, that makes sense. I know there are a lot of good queuing libraries for this.